src/plugins/vis_type_timeseries/public/application/components/vis_editor.js

```diff
@@ -134,6 +134,14 @@ export class VisEditor extends Component {
     });
   };
 
+  updateModel = () => {
+    const { params } = { ...this.props.vis.clone() };
```

Comment:
not: just interesting why we need to wrap it into `{ ... }` if `clone() `method should return a new instance

Reply:
Thanks, updated! 👍